demos-europe/demosplan-core
# Improve docker setup

This PR moves the existing ansible triggered building of the containers to Dockerfiles and provides a `docker-compose.dist.yml` with a working dev environment.


### How to review/test
run `DEMOSPLAN_PATH=<path-to-dplan> docker-compose -f docker-compose.dist.yml up` to see the magic happen.


### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
